vernac/vernacControl.ml

```diff
@@ -209,6 +217,13 @@ let rec under_control ~loc ~with_local_state controls ~noop f =
 
 let ignore_state = { with_local_state = fun _ f -> (), f () }
 
+let captured_output = Buffer.create 512 (* TODO @radrow this is likely too dumb — e.g. succeed won't protect this *)
```

Comment:
Ideally I would add this to the general state, but I don't know how to access it from control commands.